lib/data_update_scripts/20220203194510_update_article_flag_without_parsing_article.rb

```diff
@@ -0,0 +1,20 @@
+module DataUpdateScripts
+  class UpdateArticleFlagWithoutParsingArticle
+    def run
+      # Query cribbed from https://dev.to/admin/blazer/queries/545-articles-containing-a-given-string-in-the-markdown?substring=cover_image%3A+
+      Article.where(main_image_from_frontmatter: false).find_each do |article|
+        has_cover_image = false
+        begin
+          fixed_body_markdown = MarkdownProcessor::Fixer::FixAll.call(article.body_markdown || "")
+          parsed = FrontMatterParser::Parser.new(:md).call(fixed_body_markdown)
+          has_cover_image = parsed.front_matter.key?("cover_image")
+        rescue StandardError
+          # Piping this to /dev/null, because we can't assume this article is processible in our
```

Comment:
Would it be worth adding a DataDog monitor for this and incrementing it here? It might be useful to track how many unprocessable articles we have cc @jgaskins

Reply:
@citizen428 Good lookin out. Suggestion added.